benefits/enrollment/templates/enrollment/index.html

```diff
@@ -92,14 +85,21 @@
     <h2>Error!</h2>
     <p>No access token configured</p>
   {% endif %}
-{% endblock paragraph_content %}
 
-{% block buttons %}
+  {% for f in page.forms %}
+    {% include "core/includes/form.html" with form=f %}
+  {% endfor %}
+{% endblock inner-content %}
+
+{% block call-to-action-button %}
   {% if payment_processor %}
+    <button type="button" class="btn btn-primary btn-lg loading" role="status" disabled>
+      {{ payment_processor.loading_text }}
+    </button>
     <div class="invisible">
```

Comment:
Why do we have this `class="invisble"` here? I know this PR didn't add it, but I see we now render the buttons inside?

Reply:
I always assumed this `invisible` class and the corresponding JS removing it (` $(".invisible").removeClass("invisible");`) was trying to prevent the user from clicking the button prematurely or something(???)

Reply:
I agree with @machikoyasuda 's understanding of why it's here.

The main button has always been inside a `<div class="invisible">`, from what I can see in Git [history](https://github.com/cal-itp/benefits/commit/50a91417cefc86213362b034d2ccbc9db9e58c3a?diff=unified#diff-d6d406916b45273ffe3a8c113900029e13605ae79091373a7f7712824e6656e5R69-R75). The `invisible` class is removed when we get a successful response from calling the PaymentProcessor's `card_tokenize_url`.

Reply:
Oh yeah! Thanks for the reminder - I had forgotten about the JS on that page.